packages/ag-charts-enterprise/src/series/radar/radarThemes.ts

```diff
@@ -30,18 +30,17 @@ const BASE_RADAR_SERIES_THEME: _ModuleSupport.SeriesModule<'radar-line' | 'radar
     },
 };
 
-export const RADAR_LINE_SERIES_THEME = _ModuleSupport.mergeDefaults(
-    { series: { strokeWidth: 2 } },
-    BASE_RADAR_SERIES_THEME
-);
+export const RADAR_LINE_SERIES_THEME: _ModuleSupport.SeriesModule<'radar-line'>['themeTemplate'] =
+    _ModuleSupport.mergeDefaults({ series: { strokeWidth: 2 } }, BASE_RADAR_SERIES_THEME);
 
-export const RADAR_AREA_SERIES_THEME = _ModuleSupport.mergeDefaults(
-    {
-        series: {
-            fillOpacity: 0.8,
-            strokeWidth: 2,
-            marker: { enabled: false },
+export const RADAR_AREA_SERIES_THEME: _ModuleSupport.SeriesModule<'radar-area'>['themeTemplate'] =
+    _ModuleSupport.mergeDefaults(
+        {
+            series: {
+                fillOpacity: 0.8,
+                strokeWidth: 2,
+                marker: { enabled: false },
+            },
         },
-    },
-    BASE_RADAR_SERIES_THEME
-);
+        BASE_RADAR_SERIES_THEME
+    );
```

Comment:
Typescript hits an issue with depth of type evaluation and required explicit typing here. I'm not sure if it is due to the recursive nature of the `ThemeParamsOperation` & `ThemeParamsLeaf` types, of it they were just one step too far.

Something to keep an eye on when we introduce this further.